config/default.js

```diff
@@ -22,6 +22,9 @@ module.exports = {
       save: 'save',
       new: 'Save Revision',
     },
+    plugins: {
+      directory: [path.join(__dirname, '../input-plugins')],
```

Comment:
There should be unit tests to make sure this works when punchcard is used as a dependency.